services/docker-compose.yml

```diff
@@ -1,5 +1,35 @@
-version: '2'
+version: '3.1'
 services:
+  postgres:
+    image: timescale/timescaledb:latest-pg12
+    restart: always
+    environment:
+      POSTGRES_PASSWORD: "password"
+      POSTGRES_USER: "avena"
+      POSTGRES_DB: "avena"
+      TIMESCALEDB_TELEMETRY: "off"
+    ports:
+      - 5432:5432
+    volumes:
+      - postgres:/var/lib/postgresql/data
+      - ./postgres/schema:/docker-entrypoint-initdb.d
+  oada-upload:
+    build:
+      context: ./oada_upload
+    restart: always
+    environment:
+      isoblue_id: "ib999"
+      oada_server_batchsize: "1000"
+      oada_server_basepath: "/bookmarks/isoblue/device-index"
+      oada_server_url: "server_url.example.com"
+      oada_server_token: "abc"
+      db_user: "avena"
+      db_host: "postgres"
+      db_database: "avena"
+      db_password: "password"
+      db_port: "5432"
+    depends_on:
+      - postgres
   kafka:
```

Comment:
What is the future of these Kafka containers? Are we using them?

Reply:
I think their future is no more, I guess that's why I never merged them into services.